### ISSUE TITLE
5.0: Unable to open zotero urls from tinymce (worked in 4.x)

### ISSUE THREAD

**Issue by jlegewie** [zotero/zotero#1163]
Hi,

I am unable to open zotero urls from the note window. For example, left clicking on the zotero url `zotero://select/items/0_DQHBAGPE` opens the little popup. When I click on "Open Link" nothing happens and I get this error in the console:

```
Security Error: Content at resource://zotero/tinymce/note.html?locale=en-US&dir=ltr may not load or link to zotero://select/items/0_DQHBAGPE
```

Version: Zotero Standalone 5.0-beta.120+3415aae
macOS Sierra 10.12.3

**Comment by mtd91429** [comment]
At @jlegewie's suggestion, I'm re-posting my observations recorded on the zotfile issue tracker: 

I noticed this behavior yesterday and posted to the forums
[https://forums.zotero.org/discussion/64082/5-0-beta-links-are-being-deleted-in-notes](https://forums.zotero.org/discussion/64082/5-0-beta-links-are-being-deleted-in-notes
)

I definitely could be wrong, but I think that the link removal is occurring during the sync process and is unrelated to TinyMCE. 

1. Notes that aren't altered continue to contain Zotero links generated before the 5.0 upgrade. They only disappear after a change has been made to the note (and with a delay).

1. I installed the latest build of zotero into a nightly firefox and didn't sign in. In this context, the link deletion doesn't occur. 

1. I also tested this on my own reference database using the most recent beta build (`5.0-beta.120+3415aae`). The links are deleted with a delay after inserting them into a note or after altering a note. However, they are immediately deleted if you push a manual sync. After I unlinked my account, the links don't disappear.

Finally, if you look at the debugging log I posted [here](https://gist.github.com/mtd91429/970cede8bbfdd6e639f2fec204927845#file-zotero_debug_vanishinglinks-txt-L211-L287), the href is present up until the HTTP POST is performed.

I've been looking through the syncing files https://github.com/zotero/zotero/tree/master/chrome/content/zotero/xpcom/sync in Zotero trying to determine what is the origin, but haven't any luck (the origin may be beyond my knowledgebase, or my hypothesis is incorrect).

**Comment by dstillman** [comment]
@mtd91429: Yes, that's a [separate](https://github.com/zotero/dataserver/issues/35) (server-side) issue.

**Comment by mtd91429** [comment]
@dstillman I realize that this issue was closed with the commit ae47ae2. Perhaps it would be better to open a new issue, but I'm not sure. 

Anyway, I posted in the forums a description of what I believe is a bug related to this issue (and the aforementioned commit). Here is a [link](https://forums.zotero.org/discussion/64257/5-0-beta-zotero-select-links-only-partially-work-after-ae47ae2#latest) to that post. 

I also uploaded a bug report to (ID: D87724947) and pasted this to a gist [link](https://gist.github.com/mtd91429/3ce12c91233b7b6428de0fc50ed0a7c2#file-zotero-select-debug-log-txt-L59-L67)
